khalil.md

```diff
@@ -0,0 +1,42 @@
+# Hi, I'm Khalil! 👋
+
+## I have experience with:
+
+- Javascript: React, Node.js, Express, ES6
+
+- Databases:
+  SQL
+
+- Styling:
+  CSS3, SCSS
+  General Tools:
+
+- Git, Webpack
+
+## When it comes to my life, it's at its best when it involves:
+
+- 💻 Code
+
+- 🎨 Art
+
+- 🤝 Community
+
+## I'm an avid learner. Here's what im working on:
+
+- 🌱 I'm focusing on React.
+
+- 🛠 I'm also in the process of fixing up my personal website: [www.khalildev.com](www.khalildev.com)
+
+- 🐶 My pet project is a flashcard app, designed to help me study and reinforce my coding knowledge.
+
+- 💬 I love talking about programming. Contact me at any time.
+
+- 📫 How to reach me: Crenshaw.Khalil@Gmail.com
+
+## More about me:
+
+- I love digital art.
+
+- I have a knack for crafting things, and I'm also a skilled welder.
+
+- My favorite thing to do when traveling is trying the local foods.
```

Comment:
Looks cool, but I'm not sure that this should be in your project file.  I think this is supposed to be in its own repo.

Reply:
You are right. This is practice i'll remove this.

Reply:
It is now removed.